### PR TITLE
Try CircleCI orb for Avalon 6.x

### PR DESCRIPTION
**Pull request by phuongdh**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            - yarn-cache
+
+      - run: yarn
+
+      - save_cache:
+          key: yarn-cache-{{ arch }}-{{ .Branch }}-{{ checksum "yarn.lock" }}
+          paths:
+            - .cache/yarn
+
+      - run:
```
Comment (cjcolvar) [inline .circleci/config.yml]: I think this should all be handled in `samvera/code_climate_before_build` or at least that was my intention

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+          command: |
+            bundle install --with aws development test postgres --without production --path=vendor/bundle --jobs=4 --retry=3
+            bundle exec rake db:migrate
+
+      - save_cache:
+          key: gem-cache-v1-{{ .Branch }}-{{ checksum "Gemfile.lock" }}-<< parameters.ruby_ver >>
+          paths:
+            - vendor/bundle
+
+      - restore_cache:
```
Comment (cjcolvar) [inline .circleci/config.yml]: This yarn install looks really ripe to kick out to the orb after the build is passing.

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+          command: |
+            curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+            chmod +x ./cc-test-reporter
+            ./cc-test-reporter before-build
+
+      - samvera/code_climate_before_build
+
+      - samvera/parallel_rspec
+
+      - samvera/code_climate_after_build
```
Comment (cjcolvar) [inline .circleci/config.yml]: I'm not seeing this fully run in the latest circleci build and the "uploading test results" and "uploading artifacts" steps oddly running twice.
https://circleci.com/gh/avalonmediasystem/avalon/19?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Comment (phuongdh) [reply .circleci/config.yml]: @cjcolvar the rspec command already has "uploading test results" and "uploading artifacts" steps. Does codeclimate really need these?

**Pull request merged**